src/bootstrap/dist.rs

```diff
@@ -2214,6 +2347,7 @@ impl Step for HashSign {
         cmd.arg(addr);
         cmd.arg(builder.package_vers(&builder.release_num("cargo")));
         cmd.arg(builder.package_vers(&builder.release_num("rls")));
+        cmd.arg(builder.package_vers(&builder.release_num("rust-analyzer/crates/rust-analyzer")));
         cmd.arg(builder.package_vers(&builder.release_num("clippy")));
         cmd.arg(builder.package_vers(&builder.release_num("miri")));
         cmd.arg(builder.package_vers(&builder.release_num("rustfmt")));
```

Comment:
Seems like these can be refactored to use `xxx_package_vers()` family of helpers? Which probably should be left for another PR. 

I am not too happy with rust-analyzer being and odd one out here, but I am not sure how to make this better -- ra is a workspace, so we have to fetch version from the right crate.